src/app/util/basic-types.h

```diff
@@ -32,7 +32,10 @@ typedef uint8_t CHIPEndpointId;
 typedef uint16_t CHIPClusterId;
 typedef uint16_t CHIPAttributeId;
 typedef uint16_t CHIPGroupId;
-
+typedef uint16_t CHIPEventId;
```

Comment:
these should also manifest in the `chip` namespace without the `CHIP` prefix, see below

Reply:
Done, thanks